### PR TITLE
PTRENG-4591 import tests local repo, fix import issues

### PR DESCRIPTION
**Pull request by danielmkn**
- Fixed import issue for `artifactory_remote_docker_repository` and `artifactory_remote_helm_repository`, where `external_dependencies_patterns` attribute was non-importable. 
- Renamed attribute in `artifactory_remote_maven_repository` to remove duplication. 
- Added import tests and tests for the modified fields.

### CODE REVIEW

File: docs/resources/remote_helm_repository.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ The following arguments are supported, along with the [common list of arguments
...
 * `url` - (Required) The remote repo URL.
 * `helm_charts_base_url` - (Optional) No documentation is available. Hopefully you know what this means.
-* `external_dependencies_enabled` - (Optional) When set, external dependencies are rewritten.
-* `external_dependencies_patterns` - (Optional) An Allow List of Ant-style path expressions that specify where external
-  dependencies may be downloaded from. By default, this is set to ** which means that dependencies may be downloaded
-  from any external source.
+* `external_dependencies_enabled` - (Optional) When set, external dependencies are rewritten. `External Dependency Rewrite` in the UI.
+* `external_dependencies_patterns` - (Optional) An allow list of Ant-style path patterns that determine which remote VCS roots Artifactory will
+  follow to download remote modules from, when presented with 'go-import' meta tags in the remote repository response.
+  Default value in the UI is empty. This attribute must be set together with `external_dependencies_enabled = true`.
```
Comment (alexhung) [inline docs/resources/remote_helm_repository.md:38]: We should mirror the text from the Docker repo.

File: docs/resources/remote_maven_repository.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ resource "artifactory_remote_maven_repository" "maven-remote" {
   fetch_sources_eagerly              = false
   suppress_pom_consistency_checks    = false
   reject_invalid_jars                = true
-  metadata_retrieval_timeout_seconds = 120
+  metadata_retrieval_timeout_secs = 120
```
Comment (alexhung) [inline docs/resources/remote_maven_repository.md:19]: Nitpick: alignment of the `=` with previous attributes.

**Review by alexhung — APPROVED** [review]
LGTM. Couple of minor, non-blocking suggestions

File: docs/resources/remote_helm_repository.md (hunk 1 above)
Comment (danielmkn) [reply docs/resources/remote_helm_repository.md:38]: The default values are different for Helm and Docker repositories. For Docker it's `[**]`, for Helm it's empty.

**Review by alexhung — APPROVED** [review]

**Review by Turhsus — APPROVED** [review]
LGTM

**Pull request merged**